### PR TITLE
Code Quality: Files.App/Helpers - convert into constant.

### PR DESCRIPTION
**Pull request by gumbarros**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Part of #14999 

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Click settings button ...

### CODE REVIEW

File: src/Files.App/Helpers/NaturalStringComparer.cs [hunk 1]
```diff
@@ -9,20 +9,20 @@ namespace Files.App.Helpers
...
+		public const Int32 NORM_IGNOREWIDTH = 0x00020000;
+		public const Int32 NORM_LINGUISTIC_CASING = 0x08000000;
+		public const Int32 SORT_STRINGSORT = 0x00001000;
+		public const Int32 SORT_DIGITSASNUMBERS = 0x00000008;
 
-		public static readonly String LOCALE_NAME_USER_DEFAULT = null;
-		public static readonly String LOCALE_NAME_INVARIANT = String.Empty;
-		public static readonly String LOCALE_NAME_SYSTEM_DEFAULT = "!sys-default-locale";
+		public const String LOCALE_NAME_USER_DEFAULT = null;
+		public const String LOCALE_NAME_INVARIANT = "";
```
Comment (gumbarros) [inline src/Files.App/Helpers/NaturalStringComparer.cs:24]: `String.Empty` is not constant, that's why I changed to `""`

**Review by 0x5bfa — APPROVED** [review]
lgtm

File: src/Files.App/Helpers/Environment/SoftwareHelpers.cs [hunk 2]
```diff
@@ -10,8 +10,8 @@ internal static class SoftwareHelpers
 	{
 		public static bool IsVSCodeInstalled()
 		{
-			string registryKey = @"Software\Microsoft\Windows\CurrentVersion\Uninstall";
-			string vsCodeName = "Microsoft Visual Studio Code";
+			const string registryKey = @"Software\Microsoft\Windows\CurrentVersion\Uninstall";
+			const string vsCodeName = "Microsoft Visual Studio Code";
```
Comment (yaira2) [inline src/Files.App/Helpers/Environment/SoftwareHelpers.cs:14]: Should these be moved out of the field?

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**